src/internal/observable/onErrorResumeNext.ts

```diff
@@ -1,18 +1,15 @@
+/** @prettier */
 import { Observable } from '../Observable';
-import { ObservableInput } from '../types';
-import { from } from './from';
-import { isArray } from '../util/isArray';
+import { ObservableInput, ObservedValueUnionFromArray, ObservedValueOf } from '../types';
 import { EMPTY } from './empty';
+import { onErrorResumeNext as onErrorResumeNextWith } from '../operators/onErrorResumeNext';
+import { isArray } from 'util';
```

Comment:
issue: This looks like a VS Code auto-import-effected problem. AFAICT, we don't have a `util` package dependency.